### PR TITLE
[login] change variable name

### PR DESCRIPTION
**Pull request by ridz1208**
Currently, on the password expiry page, if one enters the same password that he currently has,n ot errors are reported (it should be an error) even tho the code takes this use-case into account.

The cause of this is that the code never reaches the validation function when submitting a new password on the password expiry page since it gets caught in a loop.

the following link points to the line where it should not enter when resetting an expired password
https://github.com/aces/Loris/blob/19.0-dev/php/libraries/SinglePointLogin.class.inc#L311

changing the password field name will prevent entering this function. because the $_POST['password'] will not be set on line https://github.com/aces/Loris/blob/19.0-dev/php/libraries/SinglePointLogin.class.inc#L204

Basically, in the current state, when a user is on the password-expiry page (page where the user should enter a new set of passwords) the code tries to authenticate with the "new password" rather then changing it.

The issue is only when the user enters the same password as the expired one !! any new passwword entered will actually save successfully because the code never enters that if statement linked above

### CODE REVIEW

**Comment by driusan** [comment]
What's "a mysterious bug"? What does this affect and why?

**Comment by driusan** [comment]
I'm also skeptical that this should go into 19.0 if it's just one edge case, I vaguely recall the login module breaking when the field names were changed, so this should be thoroughly tested.

**Comment by ridz1208** [comment]
@driusan 
- tried being as clear as possible with the description.
- definitely a 19.0.X bugfix
- still needs work

**Comment by ridz1208** [comment]
buggy
